_posts/2020-01-30-Age-Targeting-Part1.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Increase your target audience via ML on Spark [part 1]
+subtitle: Ground truth selection
+category: data
+tags: [adtech, howto, spark]
+author: Alessio Borgheresi
+header-img: "images/Age-Targeting/true_age.jpg"
+---
+
+
+In mobile advertising it is possible to target the audience of the campaign based on some charateristics of the users.
+Here we are focusing on the age.
```

Comment:
Add a sentence about: "Some apps provide the year of birth of their users. We actually do not know, where those apps derive the age from. But often those kinds of require a registration where you would provide your year of birth. 
However, this information is only provided by a small subset of apps during the bidding process. To increase the available targeting audience we use the information we get on the small subset of devices to infer the age of other, unknown users. To achieve this, machine learning comes into place.